amd_openvx_extensions/amd_rpp/CMakeLists.txt

```diff
@@ -21,12 +21,25 @@
 cmake_minimum_required(VERSION 3.0)
 project(vx_rpp)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../amd_openvx/cmake)
 
 find_package(AMDRPP    REQUIRED)
-find_package(OpenCL    QUIET)
+if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "OPENCL")
+    find_package(OpenCL    QUIET)
+elseif(GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
+    SET(OpenCL_FOUND FALSE)
+    IF(NOT DEFINED ENV{HSA_PATH})
+        SET(HSA_PATH ${ROCM_PATH}/hsa)
+    ELSE()
+        SET(HSA_PATH $ENV{HSA_PATH})
+    ENDIF()
+    find_package(HIP QUIET REQUIRED)
+    if(HIP_COMPILER STREQUAL clang)
+        set(HIP_LIBRARY amdhip64)
+    endif()
```

Comment:
add the below code in the case clang is not available
else()
message(FATAL_ERROR "unsupported hip compiler")

Reply:
Done.